gruntfile.js

```diff
@@ -421,14 +421,22 @@ module.exports = function (grunt) {
             clickanalytics: {
                 options: {
                     urls: [
-                        './extensions/applicationinsights-clickanalytics-js/Tests/Selenium/Tests.html'
+                        'http://localhost:9001/extensions/applicationinsights-clickanalytics-js/Tests/Selenium/Tests.html'
                     ],
                     timeout: 300 * 1000, // 5 min
                     console: true,
                     summaryOnly: false,
                     '--web-security': 'false' // we need this to allow CORS requests in PhantomJS
                 }
             },
+        },
+        connect: {
+            server: {
+                options: {
+                    port: 9001,
```

Comment:
Using port 9001 (mainly so it doesn't clash with internal project extensions -- so I can run multiple project tests at the same time)